hbt/calibration/placeholder.py

```diff
@@ -0,0 +1,84 @@
+# coding: utf-8
+
+"""
+Placeholder calibrator to produce missing single trigger columns. Includes all default calibrations.
+"""
+
+from columnflow.calibration import Calibrator, calibrator
+from columnflow.calibration.cms.met import met_phi
+from columnflow.calibration.cms.jets import jec, jer
+from columnflow.production.cms.mc_weight import mc_weight
+from columnflow.production.cms.seeds import deterministic_seeds
+from columnflow.util import maybe_import
+from columnflow.columnar_util import set_ak_column
+
+from hbt.calibration.tau import tec
+
+
+np = maybe_import("numpy")
+ak = maybe_import("awkward")
+
+
+# derive calibrators to add settings
+jec_nominal = jec.derive("jec_nominal", cls_dict={"uncertainty_sources": [], "data_only": True})
+jec_full = jec.derive("jec_nominal", cls_dict={"mc_only": True})
+
+
+@calibrator(
+    uses={
+        mc_weight, jec_nominal, jec_full, jer, tec, deterministic_seeds, met_phi,
+    },
```

Comment:
Could you remove the other calibrators / producers here?
It would be good to have this calibrator standalone so that we would use it as `--calibrators default,placeholder`. This will create two outputs (one for the default and one for this placeholder calibrator), and once we have the actual trigger flags, we can drop `placeholder` and would not need to rerun `default`.

When removing the other calibrators / producers, I guess many imports etc will also be unused and can me removed.

Reply:
please see my commit below, if this is what you wanted :)